.github/workflows/build_and_push_docker_images.yaml

```diff
@@ -167,9 +167,14 @@ jobs:
           img_suffix="-elixir"
           extra_labels="org.opencontainers.image.elixir.version=${{ matrix.elixir }}"
         fi
+        extra_deps=
+        if [[ "${{ matrix.profile }}" = *enterprise* ]]; then
+          extra_deps='libsasl2-2'
```

Comment:
why is it different from the `build` script?
extra_deps='libsasl2-2,libsasl2-modules-gssapi-mit'